README.md

```diff
@@ -10,19 +10,39 @@
 
 ### How to conduct experiments
 
-#### With [`csmock`](https://github.com/csutils/csmock)
+#### With [`csmock`](https://github.com/csutils/csmock) with `csmock plug-ins`
+Formal verification of RPM packages with CBMC, Divine, and Symbiotic is available in stable Fedora releases.
 
-```bash
-$ dnf download --source <package> # get <package>.src.rpm
-$ # or
-$ koji download-build src <package> # get <package>.src.rpm
+The csmock plug-ins for these tools are still experimental and they have
+some technical limitations:
 
-$ csmock -j<n> -f -t <tool> --<tool>-add-flag '<flags>' --<tool>-timeout <n> --keep-going --no-clean ./<package>.src.rpm # add vim, if you are planning to experiment manually
-```
+- They work only for source RPM packages that contain the %check section
+  that directly or indirectly invokes the binaries produced in the %build
+  section.
 
-`--keep-going` *continue as much as possible after an error* - error can be for example `timeout`
+- The tools are known to work reliably only for programs written in C.
 
-`--no-clean` *do not clean chroot when it becomes unused* - this option is important if you'd like to run the verification by yourself
+The plug-ins can be installed on a Fedora system using the following command:
+
+    $ sudo dnf install csmock-plugin-{cbmc,divine,symbiotic}
+
+Then you can formally verify RPM packages of your choice:
+
+```bash
+$ dnf download --source ${package} # get ${package}.src.rpm
+$ # or
+$ koji download-build src ${package} # get ${package}.src.rpm
+$ simple
+$ csmock -t ${tool} ${pkg-to-verify}.src.rpm
+$ or specific
```

Comment:
`or specific` should be excluded from the code block, so that it does not look like code to be executed.

Reply:
This comment should be resolved now.